jobs/s3-notification/templates/config/logstash.conf.erb

```diff
@@ -0,0 +1,56 @@
+input {
+  <% p('l4aws.s3_notification.queues').each do | queue | %>
+    sqs {
+      <% if p('l4aws.access_key_id') %>
+        access_key_id => "<%= p('l4aws.access_key_id') %>"
```

Comment:
Might be able to drop these in favor of a simpler config since `ctl` was switched to export the default `AWS_*` variables (required by the exec'd `pull-s3`.